string face
---
string[] available_eye_directions
bool success
string info